docs/conf.py

```diff
@@ -110,7 +110,7 @@ def _skip(self, word):
 
 # add copy button to <pre> elements inside a div with class="copyable"
 copybutton_selector = "div.copybutton pre"
-copybutton_prompt_text = "\\[\\d*\\]: |\\.\\.\\.: "
+copybutton_prompt_text = "\\[\\d*\\]: |\\.\\.\\.: |\\$"
```

Comment:
I think this makes it so the space after the prompt isn't copied
```suggestion
copybutton_prompt_text = "\\[\\d*\\]: |\\.\\.\\.: |\\$ "
```